common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -190,16 +182,19 @@ def make_selection(cls, selected, children, max_count, mode):
             else:
                 raise NotImplementedError("Unsupported mode.")
             selected |= added_block_keys
+        # TODO: used for temporary logging for EDUCATOR-1290
+        course_is_mit_supply_chain = selected and "MITx+CTL" in next(iter(selected))[1]
         if course_is_mit_supply_chain:
             logger.info(
-                "EDUCATOR-1290: LibraryContentModule.make_selection executed. "
-                "valid_block_keys: {0} | selected: {1} | invalid: {2} | overlimit: {3} | added: {4} | log ID: {5}".format(
+                "EDUCATOR-1290: LibraryContentModule.make_selection executed for course: {0} "
+                "valid_block_keys: {1} | selected: {2} | invalid: {3} | overlimit: {4} | "
+                "added: {5} | log ID: {6}".format(
                     valid_block_keys,
                     selected,
                     invalid_block_keys,
                     overlimit_block_keys,
                     added_block_keys,
-                    log_id
+                    uuid4()
```

Comment:
I just realized now that we are logging one statement, this UUID is no longer necessary or at all useful. Whoops.